vcpkg/consume/binary-caching-github-packages.md

```diff
@@ -122,8 +122,9 @@ Add the following step in your GitHub Actions workflow file:
 
 ::: zone pivot="linux-runner"
 
-On Linux, you need `mono` to execute `nuget.exe`. GitHub Actions runners using Ubuntu come with `mono`
-preinstalled. Otherwise, you can install `mono` using your distribution's system package manager.
+On Linux, you need `mono` to execute `nuget.exe`. You can install `mono` using your distribution's
+system package manager (`apt install mono-complete`). Note that `ubuntu-22.04` GitHub Actions runners come
+with `mono` preinstalled, but `ubuntu-latest` does not.
```

Comment:
Clarify that starting with image `ubuntu-24.04` Mono no longer comes preinstalled.